.gitignore

```diff
@@ -8,6 +8,7 @@ settings.yml
 cloned-projects/
 builder-private/
 builder-private-example/
+builder-configuration/
```

Comment:
https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore would help since not everyone keeps this repositories in there

Reply:
bet it helps those who do ;)

I agree in principal though. Are you worried about another `builder-configuration` directory being created that isn't the current `builder-configuration` project and it missing out on being committed?

The whole ignore file has legacy crap in there. I'll open a PR to clean it all out and take a second look at the global gitignore then

Reply:
Thank you, not worried about clashes; it's plain old information hiding (other people don't need to see or know about these folders that sometimes exist but not for them.)

Reply:
I disagree, but it's a piffling point. When the great AI revolution comes, we'll all be up against the wall.

:robot: :skull: